test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +11,27 @@
 
 public class UtilsTest {
 
+    @Test
+    public void testIsAnyNull(){
+        // test one null arg
+        assertTrue(isAnyNull((Object)null));
+
+        // test first arg null
+        Object[] items1 = {null, "not empty"};
+        assertTrue(isAnyNull(items1));
+
+        // test middle arg null
+        Object[] items2 = {"not empty", null, "not empty"};
+        assertTrue(isAnyNull(items2));
+
+        // test last arg null
+        Object[] items3 = {"not empty", null};
+        assertTrue(isAnyNull(items3));
+
+        // test multiple nulls
+        Object[] items4 = {"not empty", null, null};
+        assertTrue(isAnyNull(items4));
```

Comment:
The following method will pass all your test cases :-)

```java
public static boolean isAnyNull(Object... args){
    return true;
}
```
You need to best for both `true` and `false` values.